futures-util/Cargo.toml

```diff
@@ -23,16 +23,16 @@ bench = []
 nightly = []
 
 [dependencies]
-futures-core-preview = { path = "../futures-core", version = "0.3.0-alpha.6", default-features = false }
-futures-channel-preview = { path = "../futures-channel", version = "0.3.0-alpha.6", default-features = false }
-futures-io-preview = { path = "../futures-io", version = "0.3.0-alpha.6", default-features = false }
-futures-sink-preview = { path = "../futures-sink", version = "0.3.0-alpha.6", default-features = false}
+futures-core-preview = { path = "../futures-core", version = "0.3.0-alpha.7", default-features = false }
+futures-channel-preview = { path = "../futures-channel", version = "0.3.0-alpha.7", default-features = false }
+futures-io-preview = { path = "../futures-io", version = "0.3.0-alpha.7", default-features = false }
+futures-sink-preview = { path = "../futures-sink", version = "0.3.0-alpha.7", default-features = false}
 either = { version = "1.4", default-features = false }
 slab = { version = "0.4", optional = true }
 futures = { version = "0.1", optional = true }
 tokio-executor = { version = "0.1.2", optional = true }
 pin-utils = "0.1.0-alpha.2"
```

Comment:
This version doesn't build on latest nightly, should be bumped to `0.1.0-alpha.3` to force a `Cargo.lock` update. (I haven't checked whether any other crates directly depend on it).

Reply:
Opened https://github.com/rust-lang-nursery/futures-rs/pull/1274 to update this and re-enable the minimal versions testing

Reply:
Good catch! Thanks!